pyproject.toml

```diff
@@ -8,14 +8,17 @@ packages = [{include = "openai_functools"}]
 
 [tool.poetry.dependencies]
 python = "^3.8.1"
+openai = "1.7.2"
```

Comment:
Should this not be bigger or equal then because now its fixed to 1.7.2

Reply:
good catch!